### PR TITLE
fix more_itertools in ci

### PR DESCRIPTION
**Pull request by mabdinur**
## Commit Message

Tests failing for pytest, flask, cherrypy, and celery.

more_itertools>=8.10.11 is not compatible with python 3.5 

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

**Review by majorgreys — APPROVED** [review]

**Review by brettlangdon — APPROVED** [review]

**Comment by Kyle-Verhoog** [comment]
Looks like celery, cherrpy and flask need to have it pinned too

**Review by Kyle-Verhoog — APPROVED** [review]
🤞

File: riotfile.py [hunk 1]
```diff
@@ -607,16 +608,13 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
         Venv(
             name="flask",
             command="pytest {cmdargs} tests/contrib/flask",
-            pkgs={
-                "blinker": latest,
-            },
+            pkgs={"blinker": latest,"pytest": "~=3.0", "more_itertools": "<8.11.0"},
```
Comment (Kyle-Verhoog) [inline riotfile.py:611]: I don't think we want to be pinning pytest to 3 for all versions of flask. pytest 3 is 3 years old to the day
image

Why was this change necessary?

**Review by Kyle-Verhoog — CHANGES_REQUESTED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

File: riotfile.py [hunk 2]
```diff
@@ -626,10 +627,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
                         "DATADOG_SERVICE_NAME": "test.flask.service",
                         "DATADOG_PATCH_MODULES": "jinja2:false",
                     },
-                    pkgs={
-                        "flask": ["~=0.12.0"],
-                        "pytest": "~=3.0",
-                    },
+                    pkgs={"flask": ["~=0.12.0"], "pytest": "~=3.0", "more_itertools": "<8.11.0"},
```
Comment (brettlangdon) [inline riotfile.py:630]: WHy are we not able to define `mode_itertools` dependency just once at the main Venv for flask?

Comment (Kyle-Verhoog) [reply riotfile.py:630]: imo, probably not a good idea since it's just Python 3.5 that is impacted

**Review by majorgreys — APPROVED** [review]

File: riotfile.py (hunk 2 above)
Comment (brettlangdon) [reply riotfile.py:630]: okay
